apps/voting/test/voting.js

```diff
@@ -174,189 +140,234 @@ contract('Voting App', accounts => {
             assert.equal(await voting.minAcceptQuorumPct(), 1, 'should have changed acceptance quorum')
         })
 
-        it('fails changing minimum acceptance quorum to zero', async () => {
-            return assertRevert(async () => {
-                await voting.changeMinAcceptQuorumPct(0)
-            })
-        })
-
         it('fails changing minimum acceptance quorum to greater than min support', async () => {
             return assertRevert(async () => {
                 await voting.changeMinAcceptQuorumPct(neededSupport.plus(1))
             })
         })
 
-        context('creating vote', () => {
-            let script, voteId
+    })
+
+    for (const decimals of [0, 2, 18]) {
+        context('normal token supply, ' + decimals + ' decimals', () => {
```

Comment:
We can use the [backtick syntax](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) here :)